src/Cmp/Queues/Domain/Event/DomainEvent.php

```diff
@@ -252,6 +284,7 @@ public function jsonSerialize()
         return [
             'origin'        => $this->origin,
             'name'          => $this->name,
+            'context'       => $this->context,
```

Comment:
you dont want to add attributes here?

Reply:
No, attributes are, in fact, meta data, and we are only serialising the message in here.

We don't want a field in the message with the actual meta data of the message.